<launch>
	<node pkg="octomap_server" type="octomap_server_node" name="octomap_server">
		<param name="resolution" value="0.02" />
		
		<!-- fixed map frame (set to 'map' if SLAM or localization running!) -->
		<param name="frame_id" type="string" value="world" />

		<param name="base_frame_id" type="string" value="base_footprint" />

        <param name="filter_ground" type="boolean" value="false" />

        <!-- <param name="ground_filter/distance" type="double" value="0.05" /> -->
		
		<!-- maximum range to integrate (speedup!) -->
		<param name="sensor_model/max_range" value="20.0" />
		
		<!-- data source to integrate (PointCloud2) -->
		<remap from="cloud_in" to="velodyne_points" />
	
	</node>
</launch>